src/lib/atoms/carousel/Carouselstekje.svelte

```diff
@@ -1,27 +1,97 @@
 <script>
-  export let data;
+  import { onMount } from "svelte";
+  import { writable } from "svelte/store";
   import Arrow from "../svg/arrow.svelte";
+  export let data;
+
+    onMount(() => {
+      const prevButton = document.getElementById("button-prev");
+      prevButton.addEventListener("click", () => {
+        const el = document.querySelector(".flex-wrapper-recent-stekjes");
+        el.scrollBy({ left: -1 * el.offsetWidth });
+      });
+
+      const nextButton = document.getElementById("button-next");
+      nextButton.addEventListener("click", () => {
+        const el = document.querySelector(".flex-wrapper-recent-stekjes");
+        el.scrollBy({ left: el.offsetWidth });
+      });
+    });
+
+  // let activeIndex = 0;
+  // const indicators = writable(Array(10).fill(false));
+
+  // onMount(() => {
+  //   updateIndicators();
+
+  //   const prevButton = document.getElementById("button-prev");
+  //   prevButton.addEventListener("click", () => {
+  //     activeIndex = Math.max(0, activeIndex - 3);
+  //     updateIndicators();
+  //     const el = document.querySelector(".flex-wrapper-recent-stekjes");
+  //     el.scrollBy({ left: -1 * el.offsetWidth });
+  //   });
+
+  //   const nextButton = document.getElementById("button-next");
+  //   nextButton.addEventListener("click", () => {
+  //     activeIndex = Math.min(9, activeIndex + 3);
+  //     updateIndicators();
+  //     const el = document.querySelector(".flex-wrapper-recent-stekjes");
+  //     el.scrollBy({ left: el.offsetWidth });
+  //   });
+  // });
+
+  // function updateIndicators() {
+  //   indicators.update((values) => {
+  //     values.fill(false);
+  //     for (let i = activeIndex; i < activeIndex + 3; i++) {
+  //       values[i] = true;
+  //     }
+  //     return values;
+  //   });
+  // }
+
+  onMount(() => {
+    // Als JavaScript is ingeschakeld, toon dan de carousel-elementen
+    const carouselElements = document.querySelectorAll(
+      "button, .carousel-indicator"
+    );
+    carouselElements.forEach(function (element) {
+      element.style.display = "block";
+    });
+  });
 </script>
 
 <div class="carousel-container">
```

Comment:
Waarom is dit een div element? Kan je hiervoor een semantisch element gebruiken?

Reply:
Klopt het dat er onder de caroussel veel witruimte zit?

<img width="1524" alt="image" src="https://github.com/fdnd-agency/buurtcampus-oost/assets/1061632/6049b6fd-6a57-429b-936e-80ab4f35372d">